services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -357,6 +360,9 @@ protected SafeFuture<?> initialize() {
         eventChannels.getPublisher(CombinedStorageChannel.class, beaconAsyncRunner);
     storageQueryChannel = combinedStorageChannel;
     storageUpdateChannel = combinedStorageChannel;
+    throttlingStorageQueryChannel =
+        new ThrottlingStorageQueryChannel(storageQueryChannel, 4, metricsSystem);
```

Comment:
so we'd go from 10 down to 4?

I wonder if we want 10 in the throttled instance, and headroom like 3-5 outside of it...